src/app/mainwindow/a-inventory-window/a-shopping-cart-window/DoctorOrderServices.service.ts

```diff
@@ -40,19 +40,8 @@ export class DoctorOrderServices{
 
 
 
-  createVerifiedDoctorOrder(doctorName: string,doctorEmail: string,doctorId: string ,totalAmount: number,pickupDate: string,drugId: Array<any> = [] ,drugName: Array<any> = [],drugPrice: Array<any> = [] ,drugQuantity: Array<any> = [],realQuantity: Array<any> = [] ,doctorContact: string){
-    const VerifiedDoctorOrderData  = {doctorName:doctorName ,
-                            doctorContact:doctorContact ,
-                            doctorId:doctorId ,
-                            doctorEmail:doctorEmail ,
-                            drugId:drugId,
-                            drugName:drugName ,
-                            drugPrice:drugPrice,
-                            drugQuantity:drugQuantity,
-                            realQuantity:realQuantity,
-                            totalAmount:totalAmount,
-                            pickupDate:pickupDate};
-    this.http.post(environment.backendBaseUrl + "/api/verifiedDoctorOrder",VerifiedDoctorOrderData)
+  createVerifiedDoctorOrder(id: string){
+    this.http.patch(environment.backendBaseUrl + "/api/doctorOrder/fhir/rems/" + id, {})
```

Comment:
minor comment, but we're using `HttpClient` here to make requests, but use `axios` elsewhere.  We should try to stay consistent throughout the project.

Reply:
HttpClient is angular's defualt package for network requests and is used a lot in the frontend and should probably keep being used. The backend however is run within node and is where the axis commands exist. These were the networking tools the original repo owners made before we forked this. Normally I would agree with you, but if you look at the backend as separate from the frontend and consider them to be two separate projects even though they are in the same repo, then we are fairly consistent with which we use

Reply:
makes sense